frontend/src/components/pageNotFound/pageNotFound.module.scss

```diff
@@ -0,0 +1,35 @@
+.errorPage {
+  grid-area: main;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+body {
+  background-color: #e9e9e9;
```

Comment:
Is this a particularly nice colour we might use elsewhere in the app? If so, maybe make it a variable

Reply:
I'm not even sure if it is to be honest! I'll have a look and see if we even need a bg colour here.

Reply:
Removed bg colour for now. We can add one later if needed.